packages/flutter/test/foundation/service_extensions_test.dart

```diff
@@ -26,6 +26,9 @@ class TestServiceExtensionsBinding extends BindingBase
 
   final Map<String, ServiceExtensionCallback> extensions = <String, ServiceExtensionCallback>{};
 
+  final Map<String, List<Map<String, dynamic>>> eventsDispatched =
```

Comment:
consider putting this on one long line. I don't think readability is improved by the line break and flutter doesn't require strict 80 chars.

Reply:
done.